Index: app/src/main/java/com/jmdev/app/tecnostory/data/DataController.kt
===================================================================
diff --git a/app/src/main/java/com/jmdev/app/tecnostory/data/DataController.kt b/app/src/main/java/com/jmdev/app/tecnostory/data/DataController.kt
deleted file mode 100644
--- a/app/src/main/java/com/jmdev/app/tecnostory/data/DataController.kt	(revision fdcae9d2888c0642c1a1a025299d7d16e6adb9a4)
+++ /dev/null	(revision fdcae9d2888c0642c1a1a025299d7d16e6adb9a4)
@@ -1,20 +0,0 @@
-package com.jmdev.app.tecnostory.data
-
-
-
-fun TitleController(
-    title: Int
-): Int {
-    return title
-}
-
-fun SubTitleController(
-    subTitle: Int
-): Int {
-    return subTitle
-}
-fun TextController(
-    text: Int
-): Int {
-    return text
-}
\ No newline at end of file
diff --git a/app/src/main/java/com/jmdev/app/tecnostory/ui/components/StoryScreen.kt b/app/src/main/java/com/jmdev/app/tecnostory/ui/screens/StoryScreen.kt
rename from app/src/main/java/com/jmdev/app/tecnostory/ui/components/StoryScreen.kt
rename to app/src/main/java/com/jmdev/app/tecnostory/ui/screens/StoryScreen.kt
